### ISSUE TITLE
Security update available for jws library

### ISSUE THREAD

**Issue by astrostl** [mtgred/netrunner#677]
Howdy,

`npm install` yields "npm WARN deprecated jws@0.2.6: Security update: Versions below 3.0.0 are deprecated."

Just saying :)

**Comment by meatcompute** [comment]
Is this warning still an issue? I did not run into this setting up on my machine, os x 10.10.5

**Comment by erbridge** [comment]
I didn't notice the warning, but we're still using the insecure version:

```
$  npm list jws
netrunner@0.0.1 /home/felix/dev/netrunner
└─┬ jsonwebtoken@0.4.1
  └── jws@0.2.6
```

**Comment by erbridge** [comment]
Fixed by #2011.
